gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/writer/beans/Encass.java

```diff
@@ -6,26 +6,26 @@
 
 package com.ca.apim.gateway.cagatewayexport.tasks.explode.writer.beans;
 
-import java.util.Set;
+import java.util.List;
 
 @SuppressWarnings("unused")
 public class Encass {
-    private Set<EncassParam> arguments;
-    private Set<EncassParam> results;
+    private List<EncassParam> arguments;
```

Comment:
`Set` was required to don't get repeated things when re-exporting. going back to list is not an issue?